build/src/cli.d.ts

```diff
@@ -0,0 +1,2 @@
+#!/usr/bin/env node
```

Comment:
Not sure how this happened, but we shouldn't check the `build` dir in since it's compiler output.  Mind removing this?